docs/API_reference.md

```diff
@@ -562,6 +564,36 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+#### Plenty BI related data <a name='get-bi-section'></a>
+
+##### plenty_api_get_contacts <a name='get-bi'></a>
+
+Get list of raw data files and (optionally) download raw data.
+
+[*Optional parameter*]:
+**dataName**:  restricts the search results to raw data files e.g. orders,orderItems.
+Multible raw data files can be separated by comma.
```

Comment:
~~s/Multible/Multiple/~~
I see you fixed this in another file.

Reply:
But I would really like if you could list the different possible data file types here.